conf/app.routes

```diff
@@ -1,3 +1,3 @@
 # microservice specific routes
 
-GET        /hello-world             uk.gov.hmrc.agentepayeregistration.controllers.MicroserviceHelloWorld.hello
+POST        /register             @uk.gov.hmrc.agentepayeregistration.controllers.AgentEpayeRegistrationController.register()
```

Comment:
REST resource names should be plural nouns, not verbs: [RestFul API Conventions](https://confluence.tools.tax.service.gov.uk/display/AR/RestFul+API+Conventions)